internal/server/sshd.go

```diff
@@ -144,21 +145,19 @@ func StartSSHServer(sshListener net.Listener, privateKey ssh.Signer, insecure bo
 	log.Printf("Loading authorized keys from: %s\n", authorizedKeysPath)
 	authorizedControllers, err := readPubKeys(authorizedKeysPath)
 	if err != nil {
-		log.Fatal(err)
+		log.Println("read authorizekeys fail :", err)
 	}
 
 	if _, err := os.Stat("downloads"); err != nil && os.IsNotExist(err) {
 		os.Mkdir("downloads", 0700)
 		log.Println("Created downloads directory")
 	}
 
+	conf := config.LoadConfig(filepath.Join(dataDir, "config.json"))
+
 	clients, err := readPubKeys(authorizedControlleeKeysPath)
 	if err != nil {
-		if !insecure {
-			log.Fatal(err)
-		} else {
-			log.Println(err)
-		}
+		log.Println("read authorizedControlleeKey fail :", err)
```

Comment:
This will break things that expect authorized controllee keys to be non-empty. Also, we're not removing public key auth from this project. 
This will also mean it is impossible to identify if a client is a controllable device, or a controller.

Reply:
You can still verify based on the key, but this pr has added a password-based verification method, whether it is the control end or the controlled end, we can get it in the external api

Reply:
No. You pull request removes all ability to verify a client binary by public key. Im not going to accept work that then requires work on my behalf to add. 
Additionally, as I've said in other comments, this project is not moving over to using password authentication for the client devices to connect to the server.